src/test/java/edu/harvard/iq/dataverse/util/json/BriefJsonPrinterTest.java

```diff
@@ -65,6 +69,9 @@ public void testJson_MetadataBlock() {
         assertEquals("metadata_block_name", res.getString("name"));        
         assertEquals(1, res.getInt("id"));        
         assertEquals(3, res.keySet().size());
+        } catch (MissingResourceException e) {
```

Comment:
BriefJsonPrinterTest.testJson_MetadataBlock:62 » MissingResource Can't find bu...

The above testcase throw MissingResource exception,  since BundleUtil cannot find the property file with metadata block name

mtb.setName("metadata_block_name");

so, I just handled it with try/catch

Reply:
Does this basically render the test as not being run? I'd rather not "hide" the test; either we figure out how to make it work or we may want to consider removing it (of course that is not ideal)? Did you do any research to why this error was happening (I know you sent me an e-mail asking, but I wasn't sure either)

Reply:
The reason for the error is due to the missing file "metadata_block_name.properties" , so added it and also it cleared the exception.  Please check the latest commit.